config/generic-duet3-1lc.cfg

```diff
@@ -0,0 +1,80 @@
+# This file contains common pin mappings for the Duet3 1LC. To use
+# this config, the firmware should be compiled for the SAMC21G18 with:
+# Bootloader offset of "No Bootloader"
+# Clock Reference of "25 Mhz crystal"
```

Comment:
Should mention the 12MHz crystal on 0.6 and 1.0 boards

Reply:
Good point.